src/psyclone/tests/psyir/frontend/fparser2_test.py

```diff
@@ -565,7 +567,20 @@ def test_process_declarations_kind_new_param(f2008_parser):
             "'(KIND = blah)'" in str(err.value))
 
 
-@pytest.mark.xfail(reason="Kind parameter declarations not supported - #585")
+def test_process_declarations_expression_param(f2008_parser):
+    ''' Test that process_declarations handles the kind attribute when
```

Comment:
There's no kind parameter in the Fortran being used in this test?

Reply:
Note that this xfail is for the `test_process_declarations_kind_param` test and I updated the issue number because it still fails because the KIND() intrinsic is still not recognized.

Reply:
Thanks. I think I must have got confused.